### PR TITLE
.NET SDK Resource Provider:'StorageSync': version bump and test fixes

### PR DESCRIPTION
**Pull request by anpint**
This is a follow-up to this PR which was auto-generated and merged: https://github.com/Azure/azure-sdk-for-net/pull/5596

Swagger PR for 2019-02-01 API version: https://github.com/Azure/azure-rest-api-specs/pull/5210

Contains the following changes for the StorageSync SDK:
- Update version in Microsoft.Azure.Management.StorageSync.csproj from 2.8.0-preview to 2.9.0
- Update release notes in Microsoft.Azure.Management.StorageSync.csproj
- Fix tests, and update recordings for the new API version (2019-02-01)
- Update changelog.md

### CODE REVIEW

**Comment by ankushbindlish2** [comment]
Please add spec link in description
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2019-02-01/storagesync.json

File: src/SDKs/StorageSync/Management.StorageSync/Microsoft.Azure.Management.StorageSync.csproj [hunk 1]
```diff
@@ -6,15 +6,15 @@
     <PropertyGroup>
         <PackageId>Microsoft.Azure.Management.StorageSync</PackageId>
         <Description>Provides management capabilities for Azure Storage Sync.</Description>
-        <Version>2.8.0-preview</Version>
+        <Version>2.9.0</Version>
```
Comment (ankushbindlish2) [inline src/SDKs/StorageSync/Management.StorageSync/Microsoft.Azure.Management.StorageSync.csproj]: ><Version>2.9.0</Version> [](start = 8, length = 24)

@SDK team , Please help confirm the version. we are trying to provide a stable version number.

**Review by ankushbindlish2 — APPROVED** [review]
Changes look good.

File: src/SDKs/StorageSync/Management.StorageSync/Microsoft.Azure.Management.StorageSync.csproj (hunk 1 above)
Comment (dsgouda) [reply src/SDKs/StorageSync/Management.StorageSync/Microsoft.Azure.Management.StorageSync.csproj]: Stable version here should be `3.0.0`
Please update the AssemblyInfo.cs file too

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by anpint** [comment]
@dsgouda I updated the version number in the project file, but I couldn't find a corresponding AssemblyInfo.cs to update.

**Comment by ankushbindlish2** [comment]
Create one , inspire by Storage : https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Storage/Management.Storage/Properties/AssemblyInfo.cs

**Comment by anpint** [comment]
> Create one , inspire by Storage : https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Storage/Management.Storage/Properties/AssemblyInfo.cs

Done

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**